packages/synapse-react-client/src/components/CardContainerLogic/CardContainerLogic.tsx

```diff
@@ -38,6 +38,16 @@ export interface ColumnSpecifiedLink {
   tooltipText?: string
 }
 
+/**
+ *  Used when a column value should be previewed
+ *  Currently only works in SynapseTable (not cards!)
+ */
+export interface EntityPreviewImage {
```

Comment:
New possible column configuration (not a link, but indicating that images should be shown).  Comments copied

Reply:
Consider renaming to just `EntityImage` since we aren't using the preview